### PR TITLE
blog en: add PGroonga 2.0.3 entry

### PR DESCRIPTION
**Pull request by komainu8**

### CODE REVIEW

**Comment by kou** [comment]
Thanks!!!

@yoshihara よく気づいたね！

**Comment by obache** [comment]
"highkight" is a typo of "highlight"? and 2.0.2 entry, not 2.0.3.

File: en/_posts/2017-10-10-pgroonga-2.0.3.md [hunk 1]
```diff
@@ -0,0 +1,213 @@
...
+In this case, PGroonga is about 2 times faster than textsearch.
+
+Here is a benchmark result for full text search between PGroonga and pg\_bigm:
+
+Search keywords                       | N hits      | PGroonga     | pg\_bigm
+--------------------------------------|-------------|--------------|---------
+"PostgreSQL" or "MySQL"               | About 300   | *About 2ms*  | About 49ms
+データベース(database in Japanese)    | About 15000 | *About 49ms* | About 1300ms
+テレビアニメ(TV animation in Japanese)| About 20000 | *About 65ms* | About 2800ms
+日本(Japan in Japanese                | About 530000| About 560ms  | *About 480ms*
```
Comment (yoshihara) [inline en/_posts/2017-10-10-pgroonga-2.0.3.md:48]: missing `)` ?

**Comment by komainu8** [comment]
@obache @yoshihara Thank you for your comments.
I've fixed them in pull request below.
https://github.com/groonga/groonga.org/pull/36

**Comment by yoshihara** [comment]
@komainu8 Thank you for your response and PR 😄 I commented #36. Please check it.

**Pull request merged**